### ISSUE TITLE
cannot change database

### ISSUE THREAD

**Issue by evan2579429110** [RatkoR/laravel-crate.io#38]
I tried to use a different schema, but found it was not supported。
I should have changed it to this (ratkor/laravel-crate.io/src/RatkoR/Crate/Connectors/Connector.php:77 and ratkor/laravel-crate.io/src/RatkoR/Crate/Connectors/Connector.php:102)
![image](https://user-images.githubusercontent.com/20262206/114979970-51e6ff00-9ebe-11eb-9405-32e39077450f.png)
![image](https://user-images.githubusercontent.com/20262206/114980145-9d011200-9ebe-11eb-8319-6345e65f8315.png)

![image](https://user-images.githubusercontent.com/20262206/114980048-73e08180-9ebe-11eb-8c52-9981e58c2a75.png)

**Comment by RatkoR** [comment]
Yes, you're right. Currently we connect to the default `doc` schema.

I'd welcome a PR that adds this. Just do what you did in your example but with `isset()` test (look at $port and $host). Send $database parameter to `getMultipleHostsDsn()` method because there is no `$config` variable there.

And make sure the default database is set to `doc` for backwards compatibility.
